docs/reference/components/sources/nats.cue

```diff
@@ -0,0 +1,75 @@
+package metadata
+
+components: sources: nats: {
+	title: "NATS"
+
+	features: {
+		collect: {
+			checkpoint: enabled: false
+			tls: {
+				enabled:                true
+				can_enable:             true
+				can_verify_certificate: false
+				can_verify_hostname:    false
+				enabled_default:        false
+			}
+			from: components._nats.features.collect.from
+		}
+		multiline: enabled: false
+	}
+
+	classes: {
+		commonly_used: true
+		deployment_roles: ["aggregator"]
+		delivery:      "at_least_once"
+		development:   "stable"
```

Comment:
```suggestion
		development:   "beta"
```

We've typically marked components as beta for their initial implementation until they see more real-world use.